### PR TITLE
[Foundation] Add 20 New FN Keys

### PR DESCRIPTION
**Pull request by charleszqwang**
- Also substitute 204 and 205
- Change names of all versions in config for clarity

### CODE REVIEW

**Comment by harmony-ek** [comment]
Jenkins speaking; can one of the admins verify this patch?

**Comment by xChickens** [comment]
test this please

File: internal/configs/sharding/mainnet.go [hunk 1]
```diff
@@ -9,9 +9,10 @@ import (
 const (
 	mainnetEpochBlock1 = 344064 // 21 * 2^14
 	blocksPerShard     = 16384  // 2^14
```
Comment (xChickens) [inline internal/configs/sharding/mainnet.go]: blockPerEpoch pls

**Comment by xChickens** [comment]
ok to test

**Comment by charleszqwang** [comment]
ok to test

**Comment by xChickens** [comment]
Ship it.

**Pull request merged**